src/Analyzer.PowerShellRuleEngine.UnitTests/PowerShellRuleEngineTests.cs

```diff
@@ -22,14 +24,23 @@ public static void AssemblyInitialize(TestContext context)
         }
 
         [DataTestMethod]
-        [DataRow("success.json", 0, new int[] { }, DisplayName = "Base template")]
-        [DataRow("error_without_line_number.json", 1, new int[] { 0 }, DisplayName = "Template with an error reported without a line number")]
-        [DataRow("error_with_line_number.json", 1, new int[] { 9 }, DisplayName = "Template with an error reported with a line number")]
-        [DataRow("warning.json", 1, new int[] { 0 }, DisplayName = "Template with a warning")]
-        [DataRow("repeated_error_same_message_same_lines.json", 1, new int[] { 0 }, DisplayName = "Template with an error found in multiple places, reported with the same message and line number for each")]
+        [DataRow("template_and_resource_level_results.json", 12, new int[] { 1, 1, 1, 1, 8, 14, 17, 1, 17, 17, 1, 17 }, DisplayName = "Template with errors reported in both analysis stages")]
+        // TODO add test case for error, warning (rule with severity level of warning?) and informational (also rule with that severity level?)
```

Comment:
Why did you remove them?

Reply:
Because those test cases were covering BPA code paths that no longer exist when using PSRule

Reply:
@VeraBE TODO comments true. Sorry this one skipped my memory.

We can:

- I need to update some rules in PSRule for Azure with different error levels, so we could look at adding these. or
- Log an issue assigned to me to create a custom rule for testing in this repository.